hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3287,4 +3287,13 @@
       will create intermediate directories.
     </description>
   </property>
+  <property>
+    <name>ozone.om.snapshot.cache.max.size</name>
+    <value>200</value>
+    <tag>OZONE, OM</tag>
+    <description>
+      Size of the OM Snapshot LRU cache.  This is the maximum number of open OM Snapshot RocksDb instances
```

Comment:
We should also tune down the default memory for each snapshot rocksDB instance. It may be OK if we give lower priority to reads from Snapshots compared to active Object store. We should run some experiments to check the balance between assigned memory per rocksDB vs read performance.

Reply:
Not quite following.  How would we "tune the default memory"?

Reply:
We can use rocksDB option when opening rocksDB instance corresponding to a snapshot. We can track this task under https://issues.apache.org/jira/browse/HDDS-7276. I will resolve this comment here.